celery/worker/strategy.py

```diff
@@ -30,7 +30,7 @@ def proto1_to_proto2(message, body):
         Tuple: of ``(body, headers, already_decoded_status, utc)``
     """
     try:
-        args, kwargs = body['args'], body['kwargs']
+        args, kwargs = body.get('args', ()), body.get('kwargs', {})
```

Comment:
Minor (probably will have the same effect), but the default is an empty list, rather than a tuple.